libsession/src/main/res/values/strings.xml

```diff
@@ -163,6 +163,7 @@
     <string name="cameraErrorUnavailable">Camera unavailable.</string>
     <string name="cameraGrantAccess">Grant Camera Access</string>
     <string name="cameraGrantAccessDenied">{app_name} needs camera access to take photos and videos, but it has been permanently denied. Please continue to app settings, select \"Permissions\", and enable \"Camera\".</string>
+    <string name="permissionsCameraDenied">{app_name} needs camera access to take photos and videos, but it has been permanently denied. Tap Settings -> Permissions and turn \"Camera\" on.</string>
```

Comment:
This right-angle bracket in "Settings -> Permissions" might be a nuisance - perhaps it should be `&gt;`